src/shared/navgen/nav.cpp

```diff
@@ -231,7 +231,7 @@ static void AddTri( std::vector<float> &verts, std::vector<int> &tris, vec3_t v1
 static bool SkipContent( const dshader_t *shader ) {
 	const int solidFlags = CONTENTS_SOLID | CONTENTS_PLAYERCLIP;
 
-	if ( !( shader->contentFlags & solidFlags ) )
+	if ( !( shader->contentFlags & solidFlags ) == true )
```

Comment:
Why add `== true`? It looks weird

Reply:
This was a joke trolling @bmorel, it's now removed.